de.metas.vertical.pharma/src/main/java/de/metas/impexp/product/PharmaProductImportProcess.java

```diff
@@ -220,7 +225,8 @@ protected void afterImport()
 				.addOnlyActiveRecordsFilter()
 				.filter(sqlFilter)
 				.create()
```

Comment:
Some weeks ago we talked about a problem related to records not beeing processed in the order of their PLV?
Was that addressed?

At any rate, I suggest to order by ID here, in order for the code to be more predictable

Reply:
Yes, it was; here, we just retrieve records in order to get the matched price lists